src/main/java/edu/greenblitz/tobyDetermined/commands/LED/DefaultColor.java

```diff
@@ -0,0 +1,33 @@
+package edu.greenblitz.tobyDetermined.commands.LED;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
+
+import java.awt.*;
+
+public class DefaultColor extends LEDCommand {
+
+
+    public int cnt;
+
+
+    public DefaultColor() {
```

Comment:
IIRC Java creates an empty constructor automatically and you don't need to declare it yourself.